lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -426,29 +416,6 @@ def warn_chef_config_secret_key
         WARNING
         ui.info "* " * 40
       end
-
-      # We allow the user to specify the desired architecture of Chef to install or we default
-      # to whatever the target system is.  We assume that we are only bootstrapping 1 node at a time
-      # so we don't need to worry about multipe responses from this command.
-      def set_target_architecture(bootstrap_architecture)
-        session_results = relay_winrm_command("echo %PROCESSOR_ARCHITECTURE%")
-        if session_results.empty? || session_results[0].stdout.strip.empty?
-          raise "Response to 'echo %PROCESSOR_ARCHITECTURE%' command was invalid: #{session_results}"
-        end
-        current_architecture = session_results[0].stdout.strip == "X86" ? :i386 : :x86_64
-
-        if bootstrap_architecture.nil?
-          architecture = current_architecture
-        elsif bootstrap_architecture == :x86_64 && current_architecture == :i386
-          raise "You specified bootstrap_architecture as x86_64 but the target machine is i386. A 64 bit program cannot run on a 32 bit machine."
-        else
-          architecture = bootstrap_architecture
-        end
-
-        # The windows install script wants i686, not i386
-        architecture = :i686 if architecture == :i386
```

Comment:
https://docs.chef.io/api_omnitruck.html says that the valid value of `m (architecture)` is `i386` for windows. But since the comment here says that it should be `i686`, using the same in `windows-chef-client-msi.erb`